WordPress/src/main/java/org/wordpress/android/ui/posts/FeaturedImageHelper.kt

```diff
@@ -85,6 +85,7 @@ internal class FeaturedImageHelper @Inject constructor(
             media.localPostId = localPostId
         } else {
             AppLog.e(T.MEDIA, "Upload featured image can't be invoked without a valid local post id.")
+            return false
```

Comment:
This seems fine to me. I was just wondering if the Toast message in the calling function `doMediaUriSelected` (`R.string.file_not_found`) can be misleading in this condition. Maybe we can consider to return an enum instead of a boolean to discriminate on the message to show (maybe better) or change the message to include both cases (maybe worse)? Not sure if it's worth it tough, let me know wdyt 😊.

Reply:
The "invalid post" shouldn't happen, unless we have a bug in the app so I kept the same message. But I agree it might be confusing not just for users but especially for the HEs if the user contacts them. Fixed in 215b794638496b88cbe7fb1785c8c010a5187b6e